config.json

```
@@ -1656,6 +1656,15 @@
         "prerequisites": [],
         "difficulty": 8,
         "topics": ["recursion", "searching", "trees"]
+      },
+      {
+        "slug": "zebra-puzzle",
+        "name": "Zebra Puzzle",
+        "uuid": "3a34cb49-546f-4aae-b49e-7d5b25db7ebb",
+        "practices": [],
+        "prerequisites": ["basics", "arrays", "classes", "for-loops", "maps"],
```

No need to mention `basics` as it would be a prerequisite for all exercises (same applies for basic types like `strings` in case they don't really play a major role in the exercise).

```suggestion
        "prerequisites": ["arrays", "classes", "for-loops", "maps"],
```